.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
What's the advantages of the babelrc file over the `query: { presets: ['es2015'] }` syntax un the webpack config?